src/test/java/io/vlingo/actors/plugin/mailbox/telemetry/DefaultMailboxTelemetryTest.java

```diff
@@ -42,14 +39,20 @@ public void setUp() throws Exception {
   }
 
   @Test
-  public void testThatSendRegistersACounterOnTheActorsMailbox() {
+  public void testThatSendAndPullRegistersACounterOnTheActorsMailbox() {
     telemetry.onSendMessage(message);
```

Comment:
I think "send" and "receive" are better terms. The "pull" is actually "poll" in queue lingo, and I the "pull" hides the explicit meaning.

Reply:
Renaming it should be easy enough. I completely agree that following the convention is important so I will change it ASAP.